.github/workflows/pull-request.yml

```diff
@@ -76,6 +76,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: stackhpc/s3-active-storage-compliance-suite
+          ref: update/python-deps
```

Comment:
Compliance suite [PR](https://github.com/stackhpc/s3-active-storage-compliance-suite/pull/22) has merged now so we can remove this
```suggestion
```